### PR TITLE
[DB-180] [risk=no] Top 100 charts redesign according to new screens

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Redesigned top 100 concept charts as per this design
https://projects.invisionapp.com/d/main#/console/13577480/339048498/preview

### CODE REVIEW

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 1]
```diff
@@ -232,4 +233,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     concepts.forEach(concept => concept.expanded = false);
     r.expanded = true;
   }
+  public toggleTopConcepts(domainName) {
+    if (!this.showTopConcepts[domainName] ) {
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: nit: rm space before ` )`

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 2]
```diff
@@ -61,6 +61,7 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   showGenderGraph = false;
   showAgeGraph = true;
   showSourcesGraph = false;
+  showTopConcepts = {};
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: I'm not sure about the UX of making this sticky per domain. Seems like a single boolean would make more sense.

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 3]
```diff
@@ -39,40 +39,21 @@ <h5 class="col-xs-12 box-title">
...
-                The Data Browser searches the All of Us public data for medical concepts that
-                match the keyword or code entered in the search bar.
-                The Data Browser counts how many participants have at least
-                one mention of the matching medical concepts in their records.
-                Matching medical concepts that have the highest participant counts are returned at the top of the list.</div>
-          </clr-tooltip-content>
-        </clr-tooltip>
-      </div>
-      <div *ngIf="searchText.value.length === 0" class="col-md-8 col-sm-6 col-xs-6 flex-xs-middle card-text">
-        Showing top {{items.length}} concepts for this domain.
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html]: This text and tooltip seem to still be there in the mock. Aren't they still needed?

Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.html]: Changed this as per screen. Thanks.

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts]: Done.

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts (hunk 2 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts]: makes more sense. thanks.

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 4]
```diff
@@ -232,4 +245,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     concepts.forEach(concept => concept.expanded = false);
     r.expanded = true;
   }
+  public toggleTopConcepts() {
+    if (!this.showTopConcepts) {
```
Comment (calbach) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: nit: `this.showTopConcepts = !this.showTopConcepts;`

**Comment by calbach** [comment]
Also please fix PR title formatting. Format is described here: https://github.com/all-of-us/workbench/blob/d1c57d0540c097a6a94647358fb00fb55607b87e/.github/CONTRIBUTING.md

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts (hunk 4 above)
Comment (SrushtiGangireddy) [reply public-ui/src/app/views/ehr-view/ehr-view.component.ts]: Addressed. Thanks.

**Pull request merged**